Quick Start Guide edits (#2404)
bevyengine/bevy-website: content/learn/quick-start/getting-started/setup.md

weight = 1
+++

> We know you are itching to start making games, but Bevy requires us to do a _small_ amount of setup first.

Reviewer: ```suggestion
We know you're itching to start making games, but Bevy requires us to do a _small_ amount of setup first.
```

Install Rust by following the [Rust Getting Started Guide](https://www.rust-lang.org/learn/get-started).

Once this is done, you should have the `rustc` compiler and the `cargo` build system installed in your path.

#### Rust Learning Resources

The goal of this guide is to get started learning Bevy quickly, so it won't serve as a full Rust education.
If you would like to learn more about the Rust language, check out the following resources:

- [**The Rust Book**]: The best place to learn Rust from scratch.

> - [**Rust by Example**]: Learn Rust by working through live coding examples.

Reviewer: ```suggestion
- [**Rust by Example**]: Learn Rust by exploring interactive  Rust snippets.
```
